test/pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
 asyncio_mode = auto
+asyncio_default_fixture_loop_scope = module
```

Comment:
What does this mean, and why is it suddenly needed now?

Also, this looks like a pretty important change and the commit message doesn't even mention it.

Reply:
This is a part of an incomplete change. The PR is still in draft, stay tuned :)

Reply:
I see this line is still there, so I'm still curious what it means.

Reply:
The newer pytest-asyncio wants to have this option set explicitly. test.py runs tests file-by-file, so `session`, `package`, and `module` scopes effectively are the same. I'm trying to be as close to test.py behavior as possible.